### DISCUSSION TITLE
Security Slam 2023 Participation

### DISCUSSION THREAD

**Discussion by eddie-knight** [linkerd/linkerd2#11490] [General] [answered]
Hello linkerd community!

In case you missed it, last week was the kickoff for the month-long Cloud Native Security Slam! You can read more about the event and find the kickoff webinar recording [here](https://community.cncf.io/cloud-native-security-slam/).

In the kickoff last week, it was announced that there are 5 event badges that projects can pursue. There will be swag prizes and a $100 gift card awarded for the first 200 badges that are competed— that's up to $500 per project!

To help with work prioritization, we completed a survey of CNCF end users across multiple industries, including Construction, Cybersecurity, Aerospace & Defense, Game Development & Consumer Services, Consulting, and Nuclear. 

Through this survey, end users have identified their interest in seeing security improvements to the projects they use. We've asked them to share which Security Slam goals are most interesting to them— and we've compiled the results in a hope that this will help your prioritization during the upcoming event.

While some users have not authorized us to share their name, we've still included their responses in our calculation for you. We CAN tell you that the linkerd responses included **Epic Games**.

After calculating the responses according to the interest-weight, we've found these to be the most interesting things that linkerd end users would like to see, from the five possible Security Slam badges.

1. The Mechanizer
2. The Defender
3. The Cleaner

You can read more about the success metrics and find resources to help you achieve them [here](https://github.com/security-slam/metrics/tree/main/cncf/2023).

Join us in the **#security-slam** channel on the [CNCF Slack](https://slack.cncf.io/) to ask questions and get help from the community!

**Comment by kflynn** [comment, marked as answer]
Hey @eddie-knight, many thanks for the heads-up! I don't think we'll be able to actively participate this year, but we're glad to hear what folks are interested and we're definitely putting cycles toward supply-chain security.

I'm going to close this discussion, but feel free to reopen if you feel necessary!
